src/html.js

```diff
@@ -0,0 +1,58 @@
+/* eslint jsx-a11y/html-has-lang:0 */
+
+import React from 'react'
+import PropTypes from 'prop-types'
+
+export default function HTML(props) {
+  return (
+    <html {...props.htmlAttributes}>
+      <head>
+        <meta charSet="utf-8" />
+        <meta httpEquiv="x-ua-compatible" content="ie=edge" />
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1, shrink-to-fit=no"
+        />
+        {props.headComponents}
+      </head>
+      <body {...props.bodyAttributes}>
+        {props.preBodyComponents}
+        <div
+          key={`body`}
+          id="___gatsby"
+          dangerouslySetInnerHTML={{ __html: props.body }}
+        />
+        {props.postBodyComponents}
+        <script
+          type="text/javascript"
+          src="//downloads.mailchimp.com/js/signup-forms/popup/embed.js"
+          data-dojo-config="usePlainJson: true, isDebug: false"
+        ></script>
```

Comment:
Maybe we could use Helmet to only render this on the page which needs it? (IIRC the home page)

Reply:
I'd like too, but we need to test it, to find out if it injects and executes code correctly, and that there is no problems if the code executes before html-element for it is rendered.

For now I'd like to leave it as it was in nextjs version, but maybe we should create separate task for that.